canonical/store-components
# make entire package cards clickable

## Done
- made entire package cards clickable

### QA steps
- visit https://store-components-47.demos.haus/?path=/story/charmcard--default-story
- check if the card is clickable

## Fixes
Fixes: #https://warthogs.atlassian.net/browse/WD-4305
